tests/bootstrap.php

```diff
@@ -21,7 +21,7 @@
 
 require_once __DIR__.'/../autoload.php';
 
-$tokens = new \PHP_CodeSniffer\Util\Tokens();
+new \PHP_CodeSniffer\Util\Tokens();
```

Comment:
I'd rather not change this. Quite some external standards use the PHPCS native framework to run their own sniff tests and these will generally also call on the PHPCS native test bootstrap. I have no insights into whether any of those use this variable/expect it to be available (as it is being created in the global namespace), so removing the variable could constitute a breaking change for tests in external standards.

Reply:
I think that is very unlikely that they would rely on this as a global variable. But I don't mind reverting this either.